pkg/cmd/cli/cmd/set/env.go

```diff
@@ -200,8 +202,23 @@ func RunEnv(f *clientcmd.Factory, in io.Reader, out io.Writer, cmd *cobra.Comman
 			return nil
 		})
 		if !ok {
-			skipped++
-			continue
+			ok, err = f.UpdateObjectEnvironment(info.Object, func(vars *[]kapi.EnvVar) error {
```

Comment:
Probably ConfigMap and Secrets.

On Mon, Feb 29, 2016 at 9:26 AM, Michail Kargakis <notifications@github.com>
wrote:

> In pkg/cmd/cli/cmd/set/env.go
> <https://github.com/openshift/origin/pull/7625#discussion_r54423994>:
>
> > @@ -200,8 +202,23 @@ func RunEnv(f *clientcmd.Factory, in io.Reader, out io.Writer, cmd *cobra.Comman
> >  			return nil
> >  		})
> >  		if !ok {
> > -			skipped++
> > -			continue
> > +			ok, err = f.UpdateObjectEnvironment(info.Object, func(vars *[]kapi.EnvVar) error {
>
> The goal is to move set env upstream, so we will need it.
>
> We definitely need it, I was asking if Kube had any use for it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/7625/files#r54423994>.
>